tensorflow/python/ops/variable_scope.py

```diff
@@ -692,6 +698,39 @@ def _get_single_variable(self,
     return v
 
 
+  # Initialize variable when no initializer provided
+  def _get_default_initializer(self, name, shape=None, dtype=dtypes.float32):
+    """Provide a default initializer and a corresponding value.
+
+    Args:
+      name: see get_variable.
+      shape: see get_variable.
+      dtype: see get_variable.
+
+    Returns:
+      initializer and initializing_from_value. See get_variable above.
```

Comment:
yep, `initializing_from_value` isn't documented in `get_variable`. While, `get_variable` calls `_get_single_variable` and `_get_partitioned_variable`, and both them define it. From my perspective, the comment is a little unclear, but I think the name itself is easy to understand. Do we really need to clarify here? instead of above?